psi4/driver/driver_cbs.py

```diff
@@ -855,7 +1017,9 @@ def cbs(func, label, **kwargs):
     return_wfn = kwargs.pop('return_wfn', False)
     verbose = kwargs.pop('verbose', 0)
     ptype = kwargs.pop('ptype')
-
+    scf_alpha = kwargs.pop('scf_alpha', False)
+    corl_alpha = kwargs.pop('corl_alpha', False)
```

Comment:
Throw an item about the alphas into the docstring above, I think. scf_scheme, scf_alpha, corl_scheme, corl_alpha, delta_scheme, delta_alpha

Reply:
Fixed - I'm not super confident with sphinx syntax, I only ever used it for small things, so let me know if there are some bugs.